lms/views/lti/basic_launch.py

```diff
@@ -23,37 +23,20 @@
 
 
 def has_document_url(context, request):
-    """Get if the current launch has a resolvable document URL."""
+    """
+    Get if the current launch has a resolvable document URL.
+
+    This is imported into `lms.views.predicates` to provide the
```

Comment:
This dependency between this function and the predicates is a bit confusing so nice comment here.

I guess we eventually moved to having the "one true launch view" this would be used locally.

Reply:
The other option was to register the predicate here, but seeing as they are global whether you like it not, it doesn't seem like a good idea.